SimTracker/SiPhase2Digitizer/test/BuildFile.xml

```diff
@@ -19,7 +19,3 @@
   <flags   EDM_PLUGIN="1"/>
   <flags CXXFLAGS="-g3"/>
 </library>
-<library   file="PixelTestBeamValidation.cc" name="PixelTestBeamValidation">
```

Comment:
@suchandradutta , could you please revert this validator? As we agreed, the validator will remain in this package until the code is fully validate, afterwards it will be decided if the validator should move to the proper package (Validation) or just discarded.